tests/main/install-snaps/task.yaml

```diff
@@ -0,0 +1,82 @@
+summary: Check install popular snaps
+
+manual: true
+
+environment:
+    # High Profile
```

Comment:
Well, I'll do until we implement a way to get automatically that list. I'll be working on that.